ocaml/xapi/message_forwarding.ml

```diff
@@ -3108,8 +3108,9 @@ module Forward = functor(Local: Custom_actions.CUSTOM_ACTIONS) -> struct
       let host = Db.PBD.get_host ~__context ~self:pbd in
       do_op_on ~local_fn ~__context ~host op
 
-    (* do op on a host that can view multiple SRs, if none is found, an
-       		   exception of Not_found will be raised *)
+    (** Do op on a host that can view multiple SRs, if none is found, the
```

Comment:
Needs a sentence-break: `Do op on a host that can view multiple SRs. If none is found, ...`